tests/unit/test_output_types.py

```
@@ -0,0 +1,40 @@
+import unittest
```

Flake8 (7.3.0)

[error] 'unittest' imported but unused

(F401)